neuro_evolution_ctrnn/tools/episode_runner.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+from tools.helper import set_random_seeds
+import gym
+from tools.configurations import EpisodeRunnerCfg
+
+
+class EpisodeRunner(object):
+    def __init__(self, conf: EpisodeRunnerCfg, brain_conf: object, discrete_actions, brain_class, input_space,
+                 output_size, env_template):
+        self.conf = conf
+        self.discrete_actions = discrete_actions
+        self.brain_class = brain_class
+        self.brain_conf = brain_conf
+        self.input_space = input_space
+        self.output_size = output_size
+        self.env_id = env_template.spec.id
+
+    def eval_fitness(self, individual, seed):
+        env = gym.make(self.env_id)
+        set_random_seeds(seed, env)
+        brain = self.brain_class(self.input_space, self.output_size, individual,
+                                 self.brain_conf)
```

Comment:
> 3. or simply unnecessary because the time creating the brain is neglegible in comparison to the evaluation?

das. 

Es kann sein, dass sich das ändert, wenn die brains größer werden (und sparser), aber darüber werde ich erst nachdenken wenn es soweit ist --> "premature optimization is the root of all evil"